JS/Lesson35/onur/my-app/src/components/NewRecipe/index.jsx

```diff
@@ -1,40 +1,44 @@
-import { useForm } from 'react-hook-form';
-import './styles.css';
+import { Box, Button, Checkbox, FormControlLabel, MenuItem, Stack, TextField, Typography } from "@mui/material";
+import { useForm } from "react-hook-form"
+import { StyledBox } from '../SignInPage'
+import { useTheme } from '@mui/material/styles';
+import { useRecipes, useRecipesDispatch } from "../../Modules/recipes/RecipesProvider";
+import { RecipeActionType } from "../../Modules/recipes/models";
 
 
-// const useMyHook = () => {
-//   const myData = {id: 'xxxx', name:'Data name'};
-//   const deleteMyData = () => {console.log('Deleting data...')};
+export const NewRecipe = () => {
 
-//   return [ myData, deleteMyData ];
-// }
+	const recipeContext = useRecipes();
```

Comment:
You don't really need recipe data in this component, remove console log (you can always see current data state in dev tools) and use a timestamp as an id.